src/ImuLib_cfg.cpp

```diff
@@ -0,0 +1,61 @@
+/**
+ * @file ImuLib_cfg.cpp
+ * @author Maciej Kurcius
+ * @brief 
+ * @version 0.1
+ * @date 2022-02-15
+ * 
+ * @copyright Copyright (c) 2022
+ * 
+ */
+
+#include "ImuLib_cfg.h"
+
+#define DEGREESPERSEC_TO_RADPERSEC 0.017453293
+
+TwoWire ImuWire(IMU_SDA, IMU_SCL);
+ImuDriver ImuBno(IMU_ID, IMU_ADDR_B, &ImuWire);
```

Comment:
I don't think that we should create global variables in files other than `main.cpp`. Additionally this name is quite confusing as it is used already in the class. It should be something like `g_imu_driver`